firmware/Taskfile.dist.yml

```diff
@@ -0,0 +1,78 @@
+version: '3'
+
+env:
+  GENERATOR: -G 'Unix Makefiles'
+
+tasks:
+  project_name_fix:
+    cmds:
+      - echo 
+  build:
+    cmds:
+      - |
+        read -p "Enter the build: " BUILD 
```

Comment:
The "build" directory is fixed, it should not be an input. Move it to an env variable

Reply:
I made the changes in the latest push in my branch